docs/examples.md

```diff
@@ -626,6 +626,134 @@ public class DemoApplication {
 }
 ```
 
+### HMAC calculation (Python)
+
+```python
+from encodings.utf_8 import encode
+import hashlib
+import hmac
+import json
+from ast import Bytes
+from hmac import HMAC
+
+class Item:
+
+  def __init__(self, unitPrice: int, units: int, vatPercentage: int, productCode: str, deliveryDate: str) -> None:
+    self.unitPrice = unitPrice
+    self.units = units
+    self.vatPercentage = vatPercentage
+    self.productCode = productCode
+    self.deliveryDate = deliveryDate
+
+
+class Customer:
+
+  def __init__(self, email: str) -> None:
+    self.email = email
+
+
+class RedirectUrls:
+
+  def __init__(self, success: str, cancel: str) -> None:
+    self.success = success
+    self.cancel = cancel
+
+
+class Body:
+  def __init__(self, stamp: str, reference: str, amount: int,
+    currency: str, language: str, items: list[Item], customer: Customer, redirectUrls: RedirectUrls) -> None:
+    self.stamp = stamp
+    self.reference = reference
+    self.amount = amount
+    self.currency = currency
+    self.language = language
+    self.items = items
+    self.customer = customer
+    self.redirectUrls = redirectUrls
+
+  # create toDictionary() method to customize a convertter of object class to dict type
+  def toDictionary(self) -> dict:
+    items = []
+    for item in self.items:
+      items.append(item.__dict__)
+    return dict({
+      "stamp":self.stamp,
+      "reference":self.reference,
+      "amount":self.amount,
+      "currency":self.currency,
+      "language":self.language,
+      "items":items,
+      "customer":self.customer.__dict__,
+      "redirectUrls":self.redirectUrls.__dict__
+    })
+
+
+class Crypto:
+
+  #/
+  # @param message Raw string
+  # @param secret Merchant shared secret
+  # @return
+  #/
+  @staticmethod
+  def compute_sha256_hash(message: str, secret: str) -> str:
+
+    # message must remove the whitespaces that were created during json parsing process
+    hash = hmac.new(secret.encode(), message.replace(' ','').encode() ,digestmod=hashlib.sha256)
+    return hash.hexdigest()
+
+  #/
+  # @param secret Merchant shared secret
+  # @param hParams Headers or query string parameters
+  # @param body Request body or empty string for GET request
+  # @return
+  #/
+  @staticmethod
+  def calculate_hmac(self, secret: str, hparams: dict, body: str='') -> str:
```

Comment:
hparams could be more informative, headerParams or similar. Also in comment hparams -variable is camel case but in code it is lowercase